model_compression_toolkit/core/common/mixed_precision/resource_utilization_tools/resource_utilization_data.py

```diff
@@ -247,3 +251,20 @@ def requires_mixed_precision(in_model: Any,
     is_mixed_precision |= target_resource_utilization.total_memory < total_weights_memory_bytes + max_activation_tensor_size_bytes
     is_mixed_precision |= target_resource_utilization.bops < bops_count
     return is_mixed_precision
+
+
+def _create_core_config_for_ru(core_config: CoreConfig) -> CoreConfig:
+    """
+    Create a core config to use for resource utilization computation.
+
+    Args:
+        core_config: input core config
+
+    Returns:
+        Core config for resource utilization.
+    """
+    core_config = copy.deepcopy(core_config)
+    # HMSE is not supported without GPTQ, and resource utilization doesn't care about it, so we replace it with MSE.
```

Comment:
For resource utilization, we use GPTQ disabled, and since HMSE is not supported without GPTQ we replace it with MSE.